infra/modules/providers/azure/service-plan/variables.tf

```diff
@@ -19,18 +20,18 @@ variable "service_plan_name" {
 }
 
 variable "service_plan_tier" {
-    description = "The tier under which the service plan is created. Details can be found at https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans"
-    default     = "Isolated"
+  description = "The tier under which the service plan is created. Details can be found at https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans"
+  default     = "Standard"
 }
 
 variable "service_plan_size" {
-    description = "The compute and storage needed for the service plan to be deployed. Details can be found at https://azure.microsoft.com/en-us/pricing/details/app-service/windows/"
-    default     = "S1"
+  description = "The compute and storage needed for the service plan to be deployed. Details can be found at https://azure.microsoft.com/en-us/pricing/details/app-service/windows/"
+  default     = "S1"
 }
 
 variable "service_plan_kind" {
-    description = "The kind of Service Plan to be created. Possible values are Windows/Linux/FunctionApp/App"
-    default     = "Linux"
+  description = "The kind of Service Plan to be created. Possible values are Windows/Linux/FunctionApp/App"
```

Comment:
please include the variable type for all entries in this file so we're consistent with the other variable definitions.